src/components/SearchBox.js

```diff
@@ -285,18 +293,14 @@ class SearchBox extends Component {
             }) => (
               <div {...getRootProps({ css: suggestionsContainer })}>
                 <Input
```

Comment:
I saw we have 2 `Input` & `Icon` component. What's the reason for that?

Cant we create a Wrapper and use at both the places ?